git_pile/git_pile.py

```diff
@@ -1609,36 +1609,44 @@ def cmd_reset(args):
     if not remote_pile:
         fatal("Branch %s doesn't have an upstream" % config.pile_branch)
 
-    if args.inplace:
-        branch_dir = os.getcwd()
-        if branch_dir == pile_dir:
-            fatal("In-place reset can't reset both %s and %s branches to the same commit\n"
-                  "You are probably in the wrong directory for in-place reset."
-                  % (config.pile_branch, config.result_branch))
-
-        local_branch = git("rev-parse --abbrev-ref HEAD").stdout.strip()
-        if local_branch == "HEAD":
-            local_branch = git("rev-parse --short HEAD").stdout.strip() + " (detached)"
+    if not args.pile_only:
+        if args.inplace:
+            branch_dir = os.getcwd()
+            if branch_dir == pile_dir:
+                fatal("In-place reset can't reset both %s and %s branches to the same commit\n"
+                      "You are probably in the wrong directory for in-place reset."
+                      % (config.pile_branch, config.result_branch))
+
+            local_branch = git("rev-parse --abbrev-ref HEAD").stdout.strip()
+            if local_branch == "HEAD":
+                local_branch = git("rev-parse --short HEAD").stdout.strip() + " (detached)"
+        else:
+            branch_dir = git_worktree_get_checkout_path(gitroot, config.result_branch)
+            if not branch_dir:
+                fatal("Could not find checkout of %s branch, refusing to reset.\nYou should probably inspect '%s'"
+                      % (config.result_branch, gitroot))
+            local_branch = config.result_branch
+
+        remote_branch = git_can_fail("rev-parse --abbrev-ref %s@{u}" % config.result_branch).stdout.strip()
+        if not remote_branch:
+            fatal("Branch %s doesn't have an upstream" % config.result_branch)
     else:
-        branch_dir = git_worktree_get_checkout_path(gitroot, config.result_branch)
-        if not branch_dir:
-            fatal("Could not find checkout of %s branch, refusing to reset.\nYou should probably inspect '%s'"
-                  % (config.result_branch, gitroot))
-        local_branch = config.result_branch
-
-    remote_branch = git_can_fail("rev-parse --abbrev-ref %s@{u}" % config.result_branch).stdout.strip()
-    if not remote_branch:
-        fatal("Branch %s doesn't have an upstream" % config.result_branch)
+        orig_pile = git(f"-C {pile_dir} log --oneline --abbrev-commit --no-decorate -1 ").stdout.strip()
```

Comment:
orig_pile is not used, remove this line or use orig_pile while printing command results.

Other than this LGTM.